src/app/sh-http.js

```diff
@@ -0,0 +1,25 @@
+import {Injectable} from 'angular2/core';
```

Comment:
!fix services should follow naming conventions:
file should have `.service` and class should have `Service` suffixes